package.json

```diff
@@ -52,8 +52,10 @@
     "sequelize": "^4.38.0",
     "sequelize-cli": "^4.1.1",
     "sequelize-slugify": "^0.6.1",
+    "slug": "^0.9.1",
     "swagger-jsdoc": "^3.0.2",
     "swagger-ui-express": "^4.0.0",
+    "underscore": "^1.9.1",
```

Comment:
You are not using the package `underscore` why adding it

Reply:
I have removed it.